uReplicator-Worker/pom.xml

```diff
@@ -39,6 +39,17 @@
       <artifactId>scalatest_${kafka.scala.version}</artifactId>
       <version>${scalatest.version}</version>
     </dependency>
+    <dependency>
```

Comment:
opensource users are't able to resolve this dependency.

Reply:
Published the dependency to maven central(https://search.maven.org/search?q=a:newsimpleconsumer) and also the test cases are successful, they should be able to resolve the dependency.